source/Octopus.Cli/Util/CommandSuggester.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Octopus.Cli.Util
+{
+   public static class CommandSuggester
+    {
+        public static IEnumerable<string> SuggestCommandsFor(
+            string[] words,
+            IReadOnlyDictionary<string, string[]> completionItems)
+        {
+            words = words.Except(new[] { "octo", "complete" }).ToArray();
```

Comment:
what about `Octo`?

Reply:
Ah - yes, that 'other' command. Thanks for the spot!